### DISCUSSION TITLE
SoWin.dll can't able to find

### DISCUSSION THREAD

**Discussion by samyraj42** [coin3d/coin#540] [Q&A]
SoWin.dll can't able to find

**Comment by harzner** [comment]
Hello @samyraj42 ,

I presume that you're working on Windows, probably with Visual Studio? 

Look for the directory, where your .exe is generated, and copy the SoWin.dll into that folder. I had the same problem not long ago.

Cheers,
Harzner

**Comment by VolkerEnderlein** [comment]
Hi @samyraj42 as SoWin.dll is a Dynamic link library it needs to be found at runtime. On Windows DLLs are found according to the search path order defined [here](https://learn.microsoft.com/en-us/windows/win32/dlls/dynamic-link-library-search-order). TL;DR; you need to place SoWin.dll in the folder that contains your executable, or add the folder that contains the SoWin.dll to the PATH environment variable. Hope that helps. Cheers, Volker
